auctions/README.md

```diff
@@ -0,0 +1,117 @@
+# Auctions
+
+## Auction object
+
+```
+{
+  'id': <auction-id>,
+  'seller': <userId>,
+  'item_type': <example: neelam>,
+  'initial_price': <example: 100 dinero>,
+  'bidders_and_bids': [{
+    bidder: <userId>,
+    last_bid: <number>,
+    timestamp: <unix-timestamp>
+  }],
+  'highest_bidder': <userId>,
+  'highest_bid': <example: 200 dinero>,
+  'start_time': <unix-timestamp>,
+  'duration': <unix-timestamp>,
+  'status': <active or expired>
+}
+```
+
+## **Requests**
+
+|                 Route                    |             Description              |
+|:----------------------------------------:|:------------------------------------ |
+|       [GET /auctions](#get-auctions)     | Returns all available auctions data  |
+|      [POST /auctions](#post-auctions)    | Creates a new auction                |
+| [PATCH /auctions/:id](#patch-auctionsid) | Update the bid by user               |
+
```

Comment:
Add a new API
GET /auctions/:id - will return all the bidders and their bids for a given auction